testing/cli/test_cli_sct_qc.py

```diff
@@ -17,3 +19,100 @@ def test_sct_qc_no_checks():
                       '-s', sct_test_path('t2', 't2_seg-manual.nii.gz'),
                       '-p', 'sct_deepseg_sc',
                       '-qc-dataset', 'sct_testing_data', '-qc-subject', 'dummy'])
+
+
+# custom label to str mapping for totalspineseg
+custom_label_json = {
+    12: 'C2',
+    13: 'C3',
+    14: 'C4',
+    15: 'C5',
+    16: 'C6',
+    17: 'C7',
+    21: 'T1',
```

Comment:
Maybe it would be simpler if `totalspineseg` actually generated this file in the output (as e.g. `info_label.json` or something). That way, the user doesn't have to generate this mapping themselves, and instead already has access to it immediately?

Reply:
Hm. This is actually a little tricky, as `sct_deepseg` is currently written to only copy over image files (which we preprocess).

Since this is a very "internal" feature for now (i.e. `sct_label_vertebrae` used for `sct_deepseg` QC), maybe we could merge this PR as-is for now (just to allow @NathanMolinier to generate the desired QC reports), then discuss external usage when we support automatic QC generation for `sct_deepseg totalspineseg` (likely for 7.2).